chrome_extension/background/event.js

```diff
@@ -124,7 +116,7 @@ mooltipassEvent.onSaveSettings = function(callback, tab, settings) {
 	mooltipassEvent.onLoadSettings();
 }
 
-mooltipassEvent.onGetStatus = function(callback, tab) {
+mooltipassEvent.onGetStatus = function(callback, senderTab, tab) {
```

Comment:
ok, could you therefore modify mooltipassEvent.onMessage to check if the sender.tab is not the extension (see https://developer.chrome.com/extensions/messaging) and overwrite this argument if so?
I'm asking you this because in the current implementation, any content script could ask the blacklist status of any other opened tab.